data-prepper-plugins/peer-forwarder/src/test/java/com/amazon/dataprepper/plugins/processor/peerforwarder/PeerForwarderConfigTest.java

```diff
@@ -16,7 +16,7 @@ public void testBuildConfig() {
         final int testNumSpansPerRequest = 2;
         final int testTimeout = 300;
         final HashMap<String, Object> settings = new HashMap<>();
-        settings.put(PeerForwarderConfig.DATA_PREPPER_IPS, testPeerIps);
+        settings.put(PeerForwarderConfig.STATIC_ENDPOINTS, testPeerIps);
```

Comment:
Should we add a test case in this test for DNS hostname lookup as well?

Reply:
I've got one more follow-up PR after this one, will organize this test file a bit there.